src/App/Pages/reset.password.page.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { sendResetPassword } from '../../Redux/Actions/reset.password.action';
+import Header from '../partials/header';
+
+class ResetPassword extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			password: '',
+			confirmPassword: '',
+			passwordMatch: true,
+		};
+		this.handleSubmit = this.handleSubmit.bind(this);
+		this.showMessage = this.showMessage.bind(this);
+		this.handleChangeF = this.handleChangeF.bind(this);
+		this.handleChangeS = this.handleChangeS.bind(this);
+	}
+	handleSubmit(event) {
+		event.preventDefault();
+		const { password, confirmPassword } = this.state;
+		this.setState({
+			passwordMatch: password === confirmPassword,
+		});
+		this.props.sendResetPassword({ password, confirmPassword }, this.props.match.params.token);
+	}
+	handleChangeF(event) {
+		this.setState({
+			password: event.target.value,
+		});
+	}
+	handleChangeS(event) {
+		this.setState({
+			confirmPassword: event.target.value,
+		});
+	}
+	showMessage() {
+		const { resetPasswordStatus } = this.props;
+		if (resetPasswordStatus === 'not_started') {
+			return <br />;
+		}
+		if (!this.state.passwordMatch) {
+			return (
+				<div className='alert alert-info' role='alert'>
+					passwords provided must be the same
+				</div>
```

Comment:
@urbanishimwe great job but we don't need to create new messages here since the message is specified in the backend api